scripts/run_query_example.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
I think we should rename this, and also rename the `query_client.py`. The name should indicate that this is an example with a static set of videos, so `static_queries.py` and `run_static_queries_example.sh` would be good.

This can be a separate PR though, no need to change anything here.

Reply:
Tracked in #20, closing this.